app/handlers/clap.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const claps = require(__base + "/app/modules/posts/claps");
+
+module.exports.increaseClaps = async (req, res) => {
+  try {
+    let result = await claps.validatePost(req.body.post_id);
+    let clapped = await claps.increaseClaps(result[0]);
+    res.status(200).json({ totalLikes: clapped });
```

Comment:
not this. response module and send success 200 with all post data. not just total claps